ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -94,6 +94,7 @@ do
   cp "${item}" "${dir}/tls/${manifest##tls_}"
 done <   <( find "${SHARED_DIR}" \( -name "tls_*.key" -o -name "tls_*.pub" \) -print0)
 
+echo "$(date +%s)" > "${SHARED_DIR}/CLUSTER_INSTALL_START_TIME"
```

Comment:
Isn't `echo` redundant here? i.e. can't you just do:
```shell
date +%s > "${SHARED_DIR}/CLUSTER_INSTALL_START_TIME"
```
I see that the existing code uses the echo pattern, but unless I'm wrong (which is very possible, especially with bash) I'm surprised shellcheck allows it.

Reply:
I tried to lookup existing code to replicate it and it had both these patterns. I think echo is redundant too now and I'll remove it (if shellcheck allows me to).